opendata_module/opmon_opendata/api/postgresql_manager.py

```diff
@@ -62,21 +63,10 @@ def get_data_cursor(
             order_by_str = self._get_order_by_string(order_by, subquery_name)
             limit_str = self._get_limit_string(cursor, limit)
 
-            cursor.execute(
-                ("SELECT {selected_columns} FROM (SELECT * "
-                 "FROM {table_name} {request_in_date_constraint}) as {subquery_name} {other_constraints}"
-                 "{order_by} {limit};").format(
-                    **{
-                        'selected_columns': selected_columns_str,
-                        'table_name': self._table_name,
-                        'request_in_date_constraint': request_in_date_constraint_str,
-                        'other_constraints': other_constraints_str,
-                        'order_by': order_by_str,
-                        'limit': limit_str,
-                        'subquery_name': subquery_name}
-                )
-            )
-
+            query = ("SELECT %s FROM (SELECT * FROM %s %s) as %s %s %s %s;")
+            params = (selected_columns_str, self._table_name, request_in_date_constraint_str, subquery_name,
+                      other_constraints_str, order_by_str, limit_str)
+            cursor.execute(query, params)
```

Comment:
From my experience - when there are more than 3-4 arguments then keyword parameters should be used:
`cur.execute("""
    INSERT INTO some_table (id, created_at, updated_at, last_name)
    VALUES (%(id)s, %(created)s, %(created)s, %(name)s);
    """,
    {'id': 10, 'name': "O'Reilly", 'created': datetime.date(2020, 11, 18)})`

Reply:
Done! Thanks